### ISSUE TITLE
Avoid creating redundant tabs

### ISSUE THREAD

**Issue by 161134050** [fourtf/chatterino2#993]
When joining a channel that you're already in, don't join it, and instead, switch to that tab

**Comment by apa420** [comment]
Having a channel open multiple times in different tabs is a feature of chatterino, it allows you to have different layouts depending how you want to use chat

**Comment by savabozo** [comment]
<details>
  <summary>Could have smth like this</summary>
  
![2019-05-11_13-00-27](https://user-images.githubusercontent.com/47025788/57567611-18154000-73ed-11e9-9f76-9070aff18ff2.png)

</details>

**Comment by notheowner** [comment]
Maybe like a small message that tells you, "Hey you already have that tab open btw :)" or something along those lines.

**Comment by 161134050** [comment]
Indeed, sometimes, when joining a channel I think "oh I think i've opened this chat before somewhere", so perhaps with an option, it should just switch to that tab instead of joining it.
